### ISSUE TITLE
Could possible to separate podcast downloaded file in a individule directory?

### ISSUE THREAD

**Issue by wswv** [owncloud/music#968]
Hi, the APP in nextcloud is great, I don't need to docker other app as streamer again, thanks!
After I subscribe some podcast,  I checked music folder and found all these music files stored in root directory of the music directory. it mix many of files together, could it possible to separate it? like podcast in a directory named podcast below music, thanks!

**Comment by paulijar** [comment]
Hi. I find this issue somewhat puzzling. The thing is, the Music app doesn't download or store any podcast audio. It acts only as a hub, and the audio feeds are streamed directly from the podcast provider to your browser or Subsonic client. So they should never get stored to your Nextcloud server. Is it possible that you have used some other software which has stored those files there?

**Comment by flossgeek** [comment]
Is it possible to specify a directory on the nextcloud server to feed the podcasts, instead of streaming directly from the podcast provider? I'm quite interested in this feature.

**Comment by paulijar** [comment]
@flossgeek
Nope, there's no such feature. But why would you want to do that? You can't use Nextcloud without having a network access, and if you have the network access, then you can stream directly from the original source.

**Comment by flossgeek** [comment]
That way you could apply something like sponsorblock on it before listening. And you could add audiobooks or other sources without rss feed.

**Comment by JoshuaPettus** [comment]
Question,  how would would this be useful for audio books?   I import them as mp3 files as an album like any other music file and it shows up in my album list.   Is there a potential better way?

**Comment by flossgeek** [comment]
Players like DSub don't allow increasing the speed of music, but do allow increasing the speed of podcasts. If you use this feature, it would be useful for audiobooks too.

**Comment by paulijar** [comment]
I do see the playback speed control on DSub also when playing any album, and it works as expected.

**Comment by jordanbancino** [comment]
> [@flossgeek](https://github.com/flossgeek) Nope, there's no such feature. But why would you want to do that? You can't use Nextcloud without having a network access, and if you have the network access, then you can stream directly from the original source.

I too would be interested in this feature, as a way to automatically back up podcasts in the rare event that the podcast goes offline, then you don't lose access to your podcasts.

The assumption that you have to have network access to use Nextcloud is also false, it is very possible to access Nextcloud via a local network. If that network experiences internet downtime, the ability to still stream saved podcasts from the Nextcloud server would be nice.

**Comment by jordanbancino** [comment]
@DeepDiver1975 If I or somebody else were to open a pull request implementing this functionality, would it be accepted or is the feature considered out of scope for this app?

**Comment by paulijar** [comment]
@DeepDiver1975 I now reopened all the 120+ Music app issues you closed. If it bothers you to have the open issues here under the owncloud org, then maybe you would want to transfer the ownership of this repository to me and I could maintain it elsewhere?
